repo/nirepo.yml

```diff
@@ -0,0 +1,1783 @@
+                  example: path/to/feed
+              feedIds:
+                type: array
+                items:
+                  type: string
+                  example: 5c62bf36781e2b1c7c57daf4
+      responses:
+        202:
+          $ref: '#/responses/JobIdResponse'
+        400:
+          $ref: '#/definitions/Error'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/packages:
+    get:
+      x-ni-privilege: Read
+      x-ni-operation: getPackages
+      tags: [packages]
+      summary: Get all packages
+      description: Gets all of the packages in the service's package pool.
+      parameters:
+        - in: query
+          name: omitAttributes
+          type: boolean
+          description: If true, the "Attributes" dictionary for the package will be omitted.
+          default: false
+        - in: query
+          name: omitFeedReferences
+          type: boolean
+          description: If true, the array of feed references will be omitted from each package.
+          default: false
+        - in: query
+          name: returnUnreferencedOnly
+          type: boolean
+          description: A query parameter that determines if only packages without referencing feeds should be returned.
+          default: false
+      responses:
+        200:
+          description: The request has succeeded.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Package'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/packages/{id}:
+    parameters:
+      - in: path
+        name: id
+        description: The id of this resource.
+        type: string
+        required: true
+        x-example: "5c5d49f2781e2b2cd045267b"
+    get:
+      x-ni-privilege: Read
+      x-ni-operation: getPackages
+      tags: [packages]
+      summary: Get package details
+      description: Get package details.
+      parameters:
+        - in: query
+          name: omitAttributes
+          type: boolean
+          description: If true, the "Attributes" dictionary for the package will be omitted.
+          default: false
+          required: false
+        - in: query
+          name: omitFeedReferences
+          type: boolean
+          description: If true, the array of feed references will be omitted from the package.
+          default: false
+          required: false
+      responses:
+        200:
+          description: |
+            The request has succeeded. - The requested Package object. Not all of the fields on package objects will be available depending on the package.
+            Not all fields will have content because not all packages will store this information.
+            Do not rely on all of these fields being available in any application as some may be null, and empty string, or not exist of the object.
```

Comment:
"as some may be null, empty, or nonexistent."? If this is inaccurate, please let me know so I can iterate on it.